view/frontend/templates/product/list.phtml

```diff
@@ -0,0 +1,134 @@
+<?php
```

Comment:
I'm not a big fan of overriding the list template.

But I know, it's hard to find a point of extension to add something on these blocks, it's Magento's fault, not yours...

Take some times to investigate if there is no better approach, but I'm not sure it will be easy. Don't loose too much time, I think I'll take this one by my side later.

Reply:
I thought it was not a good idea to overriding the list template but I did not find another solution !!